python/test/unit/language/test_conversions.py

```diff
@@ -341,11 +343,16 @@ def test_typeconvert_downcast(src_dtype, dst_dtype, rounding, max_repr, device):
             pytest.skip(f"{dst_dtype} downcast with RTNE rounding tests only supported on AMDGPU MI300")
 
     if is_hip():
-        if dst_dtype == 'float8e5' and rounding == 'rtne':
-            pytest.skip(f"{dst_dtype} downcast with RTNE rounding tests only supported on NVGPU with compute capability 9.0+")
-
-        if dst_dtype == 'float8e4nv' and not (src_dtype == 'float16' and rounding == 'rtne' and is_hip_mi300()):
-            pytest.skip("float8e4nv downcast tests only supported from float16, with RTNE rounding, and on AMDGPU MI300")
+        if dst_dtype == 'float8e5' and rounding == 'rtne' and not (src_dtype == 'float32' and is_hip_mi350()):
+            pytest.skip(f"{dst_dtype} downcast with RTNE rounding tests only supported from float32, and on MI350")
+
+        if dst_dtype == 'float8e4nv':
+            if not rounding == 'rtne':
+                pytest.skip("float8e4nv downcast tests only supported with RTNE rounding on AMDGPU")
+            if is_hip_mi300() and src_dtype != 'float16':
+                pytest.skip("float8e4nv downcast tests only supported from float16, with RTNE rounding on AMDGPU MI300")
+            if is_hip_mi350() and src_dtype not in ('float32'):
```

Comment:
`src_dtype != 'float32'`

Reply:
I used a tuple here because I knew I would add fp16 and bf16 soon.

Reply:
If you want to have a tuple for future cases, you'd need to write it as `src_dtype not in ('float32', )` (note the colon there) as the current way, `'f' in ('float32')` would also be `True`.